net/devif/devif_iobsend.c

```diff
@@ -53,24 +53,65 @@
  ****************************************************************************/
 
 void devif_iob_send(FAR struct net_driver_s *dev, FAR struct iob_s *iob,
-                    unsigned int len, unsigned int offset)
+                    unsigned int len, unsigned int offset,
+                    unsigned int skip)
 {
+  unsigned int copyin;
+  int ret;
+
   if (dev == NULL || len == 0 || len >= NETDEV_PKTSIZE(dev))
     {
-      nerr("devif_iob_send error, %p, send len: %u, pkt len: %u\n",
-                                          dev, len, NETDEV_PKTSIZE(dev));
+      nerr("%s error, %p, send len: %u, pkt len: %u\n",
+            __func__, dev, len, NETDEV_PKTSIZE(dev));
```

Comment:
Is __func__ available for all compilers?

Reply:
Done